### DISCUSSION TITLE
Manager does not display any data 2.4.20

### DISCUSSION THREAD

**Discussion by estherios** [Security-Onion-Solutions/securityonion#11762] [General]
I have recently installed a new distributed setup for security onion, I have a manager and a sensor. On sensor I see suricata and zeek traffic from the monitoring interface, however when I go the the manager I don't see anything on the dashboard or in Kinaba. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/103294472/e520d2c7-9a9e-472a-9017-865011820c28)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/103294472/cea8e056-1976-45da-96a0-a882bec85180)

Processes on manager 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/103294472/c97ba6c8-e83b-45f6-b91b-ca17ba44439c)

Processes on sensor 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/103294472/f257ea8a-c08d-4479-8ca0-6d85f606a927)

Both are in OK status. 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/103294472/96104c0f-989f-49b2-9fff-eb3d770c33d1)

What could I have been missing ?

**Comment by dougburks** [comment]
If you are able to add a third node, then you can add a search node and you should start seeing logs. If for some reason you can only use two nodes then you can build a new deployment with a Manager Search node (instead of just a Manager) and a sensor. 

From https://docs.securityonion.net/en/2.4/architecture.html#distributed:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/383387c4-f1d2-4784-8321-e6e2e7e0085d)
